lib/eco/incparser/test/test_lrparser.py

```diff
@@ -30,7 +30,7 @@
 """
 
 import pytest
-pytest.skip("deprecated")
+pytestmark = pytest.mark.skip
```

Comment:
What's the point of this?

Reply:
This test is deprecated. I could also just delete it I guess. Wasn't sure if this might be useful again some day.

Reply:
Up to you. I'm generally not a fan of keeping dead code around unless there's a very good reason for doing so. Dead code is always in the git history if it needs to be fished out...

Reply:
Fair enough. I'll remove it.